### ISSUE TITLE
[BUG] loading of unloaded worlds

### ISSUE THREAD

**Issue by Fusezion** [ShaneBeee/SkBee#330]
**Describe the bug**   
When attempting to load an unloaded vanilla world it fails to do so and instead creates a "copy" of the world using it's name but as a plain vanilla world

**To Reproduce**   
Please include clear steps to reproduce the behavior
```applescript
command /example:
    trigger:
        unload world "world_nether"
        wait 5 ticks
        load world "world_nether"
        wait 5 ticks
        set block at player to portal
```

**Expected behavior**   
It would load the world it's intended to without creating a "copy" using default normal world values

**Version Info:**
```
[09:14:46 INFO]: - Plugin is up to date!
[09:14:46 INFO]: --- [Server Info] ---
[09:14:46 INFO]: Server Version: git-Paper-215 (MC: 1.19.2)
[09:14:46 INFO]: Skript Version: 2.6.4
[09:14:46 INFO]: Skript Addons:
[09:14:46 INFO]: - Skacket v1.0.13
[09:14:46 INFO]: - skript-gui v1.3
[09:14:46 INFO]: - skript-yaml v1.4.1
[09:14:46 INFO]: - skript-reflect v2.3
[09:14:46 INFO]: - SkJade v1.4.2
[09:14:46 INFO]: - Skript-Advancements v1.5.2
[09:14:46 INFO]: SkBee Version: 2.7.3
[09:14:46 INFO]: SkBee Website: https://github.com/ShaneBeee/SkBee
```

**Additional context**   
Talked with Shane about this prior and said it might only be related to vanilla worlds, proper testing will be needed to ensure this is the case.

**Comment by ShaneBeee** [comment]
After much testing, I've come to find this is not an SkBee issue, but rather a quirk with how Bukkit handles vanilla dimensions.

In a nether world, this is how files are stored:
<img width="1018" alt="Screenshot 2023-01-28 at 8 19 49 AM" src="https://user-images.githubusercontent.com/20008482/215277905-949cfaa4-80ef-452d-b040-abf0f2355b7c.png">

Everything is put into the DIM-1 folder (just like vanilla MC)

When you unload the world, and re-load it, Bukkit has no clue that was supposed to be a vanilla nether world so Bukkit treats its as a whole new world:
<img width="1018" alt="Screenshot 2023-01-28 at 8 20 56 AM" src="https://user-images.githubusercontent.com/20008482/215277983-b0b38ea0-cedc-42d2-9318-7b19c5f87072.png">
As you can see here, the files are created making this a "new" world.


I did some testing with reflect and Bukkit code, first I unloaded the world, and used this to load it again:
```vb
import:
	org.bukkit.WorldCreator
	org.bukkit.World$Environment

command /test:
	trigger:
		set {_w} to new WorldCreator("world_nether")
		{_w}.createWorld()
```
This had the same effect as you mentioned.
I then used this to try the same:
```vb
import:
	org.bukkit.WorldCreator
	org.bukkit.World$Environment

command /test:
	trigger:
		set {_w} to new WorldCreator("world_nether")
		{_w}.environment(Environment.NETHER)
		{_w}.createWorld()
```
This time it worked, since I told Bukkit to treat it as a nether world.


tldr;
Not an SkBee issue, just a Bukkit quirk.

**Comment by ShaneBeee** [comment]
Closing as this did not appear to be an SkBee issue
